### PR TITLE
Pirate items for Pirates.

### PR DESCRIPTION
**Pull request by brainfood1183**
A load of new pirate themed items which should be added to the pirate ship so that would be pirates have some new items to use.
They can also be added in maintenance tunnels for crewmembers to find.

Untitled-15

What is added

- Pirate Captain Jumpsuit, also has a pirate themed accent attached to it.
- Pirate Jumpsuit, also has a pirate themed accent.
- Pirate Captains helmet complete with a built in pirate hat.
- Pirate captains ancient hardsuit, some optional protection for the pirate captain.
- Pirate captains cloak, look cool whilst you pillage and plunder.
- Pirate backpack, a pirate themed backpack more suitable than a standard backpack.
- cutlass, pirates now have a cool looking pirate sword to use.
- energy cutlass, the captain has an energy version of the cutlass for extra killing.

What will come later

- planned pirate eye patch, to look even more like a pirate
- planned boarding pistol, kind of like a mini shotgun.
- planned airlock breacher, sort of like a 1 use syndicate jaws of life.
- planned unique lazer pistol for the captain.
- new planned pirate hardsuits for the different positions aboard the pirate ship.
- planned pirate pda.

### CODE REVIEW

File: Content.Server/Speech/EntitySystems/PirateAccentSystem.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        { "yo", "Yarr" },
+        { "friend", "matey" },
+        { "crew", "me harties" },
+        { "yes", "aye" },
+        { "yes captain", "aye aye captain"},
+        { "hand hoy", "emergency" },
+        { "get ready", "batten down the hatches"},
+        { "wow", "blimey"},
+        { "pirate", "buccaneer" },
+        { "song", "sea chanty" },
```
Comment (DerSheppard) [inline Content.Server/Speech/EntitySystems/PirateAccentSystem.cs:26]: 'sea shanty', not 'sea chanty'

**Review by brainfood1183 — COMMENTED** [review]
Updated copyright info on some jsons

**Review by brainfood1183 — COMMENTED** [review]
another json copyright changed.

File: Content.Server/Speech/EntitySystems/PirateAccentSystem.cs (hunk 1 above)
Comment (brainfood1183) [reply Content.Server/Speech/EntitySystems/PirateAccentSystem.cs:26]: yea, on it

**Review by brainfood1183 — COMMENTED** [review]
changed spelling error in pirateaccentsystem.cs

**Review by brainfood1183 — COMMENTED** [review]
fixed another spelling error in the accent

**Pull request closed without merging**